File: backend/ViewModels/UserModel.cs
using System.ComponentModel.DataAnnotations;

namespace backend.ViewModels
{
    public class UserModel
    {
               [Required]
        public string name { get; set; }
        [Required]
        [StringLength(8, MinimumLength = 4, ErrorMessage = "you must specify password between4 and 8 character")]
        public string password { get; set; }
        public string isAdmin { get; set; }
    }
}

File: frontend/src/app/auth/user.model.ts
export class User {
    id: number;
    name: string;
    password: string;
    isAdmin: boolean;

    constructor(user?) {
        this.id = user ? user.id : 0;
        this.name = user ? user.name : '';
        this.password = user ? user.password : '';
        this.isAdmin = user ? user.isAdmin : false;

    }
}
